require/require.go.tmpl

```diff
@@ -1,4 +1,5 @@
-{{.Comment}}
+{{ replace .Comment "assert." "require."}}
+// Instead of returning a boolean result this function calls `t.FailNow()` on failure.
```

Comment:
The "FailNow on failure" sounds a bit strange to me

Maybe:
- calls FailNow when a test fails
- calls FailNow on first invalid assertion

you could also add it has the consequence to stop all tests execution

These are wording idea, more a functional point than a real wording